domains/certificates/Query.API/API/API.csproj

```diff
@@ -18,6 +18,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="7.0.4" />
     <PackageReference Include="Microsoft.Extensions.Http.Polly" Version="7.0.4" />
     <PackageReference Include="Serilog.AspNetCore" Version="6.0.1" />
+    <PackageReference Include="Serilog.Enrichers.Span" Version="3.1.0" />
```

Comment:
Why serilog changes?

Reply:
I added Serilog.Enrichers.Span, which ensures that we have traceid and spanid in our logs. Traceid is returned as part of the body for ValidationProblemDetails. So now if someone calling our API gets a 400 Bad Request, we can find the corresponding logs from the traceid.